specification/contosowidgetmanager/Contoso.WidgetManager/tspconfig.yaml

```diff
@@ -0,0 +1,39 @@
+parameters:
+  "python-sdk-folder":
+    default: "{project-root}/azure-sdk-for-python/"
+  "java-sdk-folder":
+    default: "{project-root}/azure-sdk-for-java/"
+  "js-sdk-folder":
+    default: "{project-root}/azure-sdk-for-js/"
+  "csharp-sdk-folder":
+    default: "{project-root}/azure-sdk-for-csharp/"
+  "service-directory-name":
+    default: "contosowidgetmanager"
+emit:
+  - "@azure-tools/typespec-autorest"
+options:
+  "@azure-tools/typespec-autorest":
+    output-file: widgets.json
+    azure-resource-provider-folder: "./data-plane"
+    emitter-output-dir: "{project-root}/.."
+    examples-directory: "./examples"
+  "@azure-tools/typespec-python":
+    package-name: "azure-contosowidgetmanager"
+    emitter-output-dir: "{python-sdk-folder}/sdk/{service-directory-name}/{package-name}"
+    package-mode: "dataplane"
+  "@azure-tools/typespec-csharp":
+    namespace: Azure.Template.Contoso
+    clear-output-folder: true
+    model-namespace: false
+    emitter-output-dir: "{csharp-sdk-folder}/sdk/{service-directory-name}/{namespace}/src"
+  "@azure-tools/typespec-typescript":
```

Comment:
```suggestion
  "@azure-tools/typespec-ts":
```

Reply:
Please notice that the reviewed package name for typescript is `@azure-tools/typespec-ts` not `@azure-tools/typespec-typescript`.

https://www.npmjs.com/package/@azure-tools/typespec-ts